src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: string = 'triangle',
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must be greater than 0');
```

Comment:
[CODE STYLE] - The error message for the Triangle constructor should be more descriptive. Instead of 'Sides must be greater than 0', consider specifying which parameter is incorrect, for example: 'Triangle side lengths must be greater than 0'.